site/docs/index.md

```diff
@@ -0,0 +1,4 @@
+---
+redirect_from: "docs/latest"
+redirect_to: "docs/v1.0.0"
```

Comment:
So we have a `latest` variable in the _config.yaml file. I'm wondering if we could include it here as `docs/{ site.latest }`, but I'm not sure. Being able to do so would mean we don't have to update the latest version in 2 spots.

Reply:
Yeah that doesn't work unfortunately. It's been asked for by the Jekyll community, but I tried it, and nope, it doesn't work.
Right now we'll have to do 1 update for the main site to point to the right docs, and then 1 minor update for this to work.

Reply:
Ah, that's unfortunate.

Here's what we had before, and it worked fairly well. Looks like it also gets to use site.latest 

https://github.com/heptio/velero/blob/gh-pages/_layouts/redirect.html

Reply:
I'll see if I can make that work here.

Reply:
Had to change it a bit, got it working. Will not work with `docs/latest`, but `/docs` will always point to the `latest` configured in `_config.yml`.